app/assets/stylesheets/application.scss

```diff
@@ -10419,4 +10419,51 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
 
 .footer {
   padding: 5rem 0;
-}
\ No newline at end of file
+}
+
+.thumbnail {
+  height: 60px;
+  width: 60px;
+}
+.header-index {
+  position: absolute;
+  top: 75px;
+  left: 20vw;
+}
+.top-margin {
+  position: absolute;
+  top: 200px;
+  left: 20vw;
+  height: 70vh;
+  overflow: scroll;
```

Comment:
This will allow the background to remain fixed while the games scoll inside of its section